source/getting_started/install_macs.md

```diff
@@ -0,0 +1,86 @@
+# Prerequisites for Macs
+
+You must have the following set up on your mac laptop:
+
+- administrator rights on your laptop
+- text editor
+- [Xcode](https://developer.apple.com/xcode/) command line interface tools [external link]
+- [Ruby](https://www.ruby-lang.org/en/) [external link]
+- [Bundler](https://bundler.io/) [external link]
+- [Middleman](https://middlemanapp.com/) static site generator [external link]
+
+You must also have a [GitHub account](https://github.com/) [external link] with access to alphagov at `https://github.com/alphagov/paas-tech-docs`.
+
+## Administrator rights on your laptop
+
+Email [sd-community@digital.cabinet-office.gov.uk](mailto:sd-community@digital.cabinet-office.gov.uk) to request administrator rights on your laptop.
+
+## Install text editor
```

Comment:
We probably don't need to tell people to use a specific text editor; we can assume they will have one of their choosing

Reply:
Have trimmed